helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -23,7 +42,7 @@ spec:
         cancel_if_cluster_status_updating: "true"
         cancel_if_cluster_status_deleting: "true"
         cancel_if_cluster_has_no_workers: "true"
-        cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
+        cancel_if_outside_working_hours: "false"
```

Comment:
Why this change ?

Reply:
Short answer: The "workingHoursOnly" variable is very counter-intuitive and should not be used.

Longer answer:
It's defined here: https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/_helpers.tpl#L44-L50
It's set to `false` on most installations,
except for `openstack` and `capz` installations where it's set to `true`.
This kind of inconsistency is a nice recipe for hard-to-debug alerts :shrug:

Reply:
I agree.
This change affects the actual behaviour for capz right ?
I don't know why we have a specific case for capz BTW...

Reply:
> This change affects the actual behaviour for capz right ?

Right.

> I don't know why we have a specific case for capz BTW...

Not sure there's a valid reason :shrug: